packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -36,6 +36,7 @@ describe("wrangler", () => {
 			  wrangler publish [script]    🆙 Publish your Worker to Cloudflare.
 			  wrangler tail [worker]       🦚 Starts a log tailing session for a published Worker.
 			  wrangler secret              🤫 Generate a secret that can be referenced in a Worker
+			  wrangler secret:bulk [json]  🗄️ Bulk upload secrets for a Worker
```

Comment:
Does this need updating to `wrangler secret:bulk <json>` too?

Reply:
Already updated based on @cameron-robey review 😄

Reply:
Also updated the snapshots 👍🏻